error.go

```diff
@@ -24,7 +25,14 @@ type Error struct {
 }
 
 func (e Error) Error() string {
-	return "mssql: " + e.Message
+	var msg strings.Builder
+	for i, err := range e.All {
+		if i > 0 {
+			msg.WriteByte('\n')
+		}
+		msg.WriteString(fmt.Sprintf("mssql: %s (%d)", err.Message, err.Number))
+	}
+	return msg.String()
 }
```

Comment:
how is this change not affecting any tests? Is there really no coverage of this call?